deploy/layered-sre-authorization/01-layered-cs-sre-admin.ClusterRole.yaml

```diff
@@ -0,0 +1,87 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: layered-cs-sre-admin
+rules:
+# CS SRE can get, list, and watch all resources
+- apiGroups:
+  - '*'
+  resources:
+  - '*'
+  verbs:
+  - get
+  - list
+  - watch
+ # CS SRE can run commands in pods
+- apiGroups:
+  - ""
+  resources:
+  - pods/exec
+  verbs:
+  - create
+# CS SRE can delete jobs and builds
+- apiGroups:
+  - batch
+  resources:
+  - jobs
+  verbs:
+  - delete
+  - deletecollection
+- apiGroups:
+  - build.openshift.io
+  resources:
+  - builds
+  verbs:
+  - delete
+  - deletecollection
+# CS SRE can manage projects
+- apiGroups:
+  - config.openshift.io
+  resources:
+  - projects
+  verbs:
+  - '*'
+- apiGroups:
+  - project.openshift.io
+  resources:
+  - projects
```

Comment:
I believe this must be granted at a cluster scope.  If so, need to create a separate ClusterRole and a ClusterRoleBinding to the `layered-cs-sre-admins` group.

Reply:
I don't really understand what needs to be done here. This is a ClusterRole?

Reply:
A ClusterRole isn't applied until you bind it to something.  To grant project and namespace I think you have to use a ClusterRoleBinding.  Using SubjectPermission CR will create RoleBindings in the matching namespaces.  You need to create a ClusterRoleBinding as a part of this PR to get the cluster scope permissions.

For example, dedicated-admin permissions are split into two ClusterRoles: dedicated-admins-cluster [1] and dedicated-admins-project [2].  A ClusterRoleBinding is created for the dedicated-admins-cluster permissions [3], a SubjectPermission is created for the dedicated-admins-project permissions [4] which results in RoleBindings in each customer namespace.

[1] https://github.com/openshift/managed-cluster-config/blob/master/deploy/rbac-permissions-operator-config/00-dedicated-admins-cluster.ClusterRole.yaml
[2] https://github.com/openshift/managed-cluster-config/blob/master/deploy/rbac-permissions-operator-config/00-dedicated-admins-project.ClusterRole.yaml
[3] https://github.com/openshift/managed-cluster-config/blob/master/deploy/rbac-permissions-operator-config/00-dedicated-admins-cluster.ClusterRoleBinding.yaml
[4]

Reply:
I thought the RBAC operator created the ClusterRoleBinding too. I'll validate this tomorrow to check. I made a change to this piece earlier in the PR and now need to retest it.

Reply:
Ok, I've tested this. The ClusterRoleBinding is created (if missing). I did lose the layered-cs-sre-admin ClusterRoleBinding though. I've re-added it to the SubjectPermission. Nice catch. Thank you.